browser-extension/src/addTournesolRecommendations.css

```
@@ -301,16 +306,18 @@ html[darker-dark-theme][dark] .tournesol_simple_button:disabled img {
 
 .time_span {
   position: absolute;
-  bottom: 0;
-  right: 0;
-  color: white;
-  background-color: var(--yt-spec-static-overlay-background-heavy);
+  bottom: 6px;
+  right: 6px;
```

(very minor) 3px seems closer to the Youtube design to my eyes
```suggestion
  bottom: 3px;
  right: 3px;
```